src/opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQDefine.cpp

```diff
@@ -198,6 +198,7 @@ UDQDefine::UDQDefine(const UDQParams& udq_params,
     }
     std::vector<UDQToken> tokens = make_tokens(string_tokens);
     this->ast = std::make_shared<UDQASTNode>( UDQParser::parse(udq_params, this->m_var_type, this->m_keyword, this->m_location, tokens, parseContext, errors) );
+    this->m_tokens = tokens;
```

Comment:
No need to to go via the local temporary `tokens`- just assign to the member variable right away.

Reply:
OK.